lib/services.sh

```diff
@@ -24,17 +24,7 @@ start_service_checker() {
   local s=1
```

Comment:
Can remove this line and hardcode `MAXRETRIES`

Reply:
yes and no

Reply:
(I don't know why MAX_RETRIES is softcoded to begin with but I have no strong feelings either way and it's orthogonal to the change I'm actually making here)